From 4ec989eed9f53d447549aad8473afbe745af7df1 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Tue, 24 Sep 2024 15:02:52 +0200
Subject: [PATCH] arm64: dts: st: fix display support on stm32mp215f-dk

Missing default-on property on panel.

Change-Id: Ia33f80097a128c3faaba6e1a7aadb402c69d8044
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/406524
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp215f-dk.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp215f-dk.dts b/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
index a9fb4ad160b7..e9edec1e6539 100644
--- a/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
@@ -155,6 +155,7 @@ panel_rgb: panel-rgb {
 		backlight = <&panel_backlight>;
 		power-supply = <&scmi_v3v3>;
 		data-mapping = "bgr666";
+		default-on;
 		status = "okay";
 
 		width-mm = <105>;
-- 
2.50.1

